daemon/cmd/daemon.go

```diff
@@ -686,6 +696,26 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	if len(restoredCIDRidentities) > 0 {
 		d.ipcache.UpsertGeneratedIdentities(restoredCIDRidentities)
 	}
+	// Upsert restored local Ingress IPs
+	nIngressIPs := 0
+	var firstErr error
+	for _, ingressIP := range oldIngressIPs {
+		_, err := d.ipcache.Upsert(ingressIP.String(), nil, 0, nil, ipcache.Identity{
+			ID:     identity.ReservedIdentityIngress,
+			Source: source.Local,
```

Comment:
Thanks for the correction, at the minimum will fix the comment, will have to see. Current code never tries to re-allocate Ingress IPs so adding a new "source" is not an immediate concern. Generally I like the suggestion on a "restored" source that would be the lowest priority, though.

Reply:
yeah I think the danger with using `Local` is that it can be difficult to flush this state out if it remains like that. Each time you restart, it would be put back in place. You also can't override it by just updating the corresponding k8s resource.

Reply:
Added the new `Restored` source.